src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -534,6 +536,8 @@ protected function initializeContainer(): void
 
     /**
      * Returns the kernel parameters.
+     *
+     * @return array<string, mixed>
```

Comment:
This can be more restrictive:
```suggestion
     * @return array<string, array|bool|string|int|float|\UnitEnum|null>
```

Reply:
Are we sure, this union is complete? What about `Stringable`?

Reply:
I took this union from 
https://github.com/symfony/symfony/blob/6fc9000504b597fbb2dc1204ef20d882838c6b59/src/Symfony/Component/DependencyInjection/ContainerInterface.php#L59

Reply:
Makes sense. 🙂